data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_abilities_class.lst

```diff
@@ -370,7 +375,7 @@ Draconic Resistance		KEY:Draconic Resistance ~ Acid				CATEGORY:Special Ability
+Breath Weapon			KEY:Draconic Bloodrager ~ Breath Weapon			CATEGORY:Special Ability	TYPE:Internal.Class Feature						VISIBLE:NO	DEFINE:BreathWeaponTimes|0	DEFINE:BreathWeaponDice|0	DEFINE:BreathWeaponDC|0																																																																																				ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Acid Cone|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Green)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Acid Line|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Black),Bloodrager Bloodline ~ Draconic (Copper)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Cold|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (White),Bloodrager Bloodline ~ Draconic (Silver)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Electricity|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Blue),Bloodrager Bloodline ~ Draconic (Bronze)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Fire Cone|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Red),Bloodrager Bloodline ~ Draconic (Gold)	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Fire Line|PREABILITY:1,CATEGORY=Special Ability,Bloodrager Bloodline ~ Draconic (Brass)																																		BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,8	BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,16	BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,20	BONUS:VAR|BreathWeaponDice|BloodlineLVL	BONUS:VAR|BreathWeaponDC|10+(BloodlineLVL/2)+CON
```

Comment:
What was wrong with this bonus statement?
max(0,(BloodlineLVL-12)/4)+1


Which leads me to the next head scratcher, why are we making three different statement bonuses instead of combining into a formula?

BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,8	BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,16	BONUS:VAR|BreathWeaponTimes|1|PREVARGTEQ:BloodlineLVL,20

Should be
BONUS:VAR|BreathWeaponTimes|1+(BloodlineLVL>=16)+(BloodlineLVL>=20)

I don't see why the first variable requires a PRExxx, the feature is added at 8th level correct? If you can't use it at least once per day, then you shouldn't have access to it, or am I missing something?